docs/templates/modules/hub.md

```diff
@@ -60,7 +60,8 @@ On this page:
 >   "type": "HubInstance",
 >   "version": "0.0.1",
 >   "learnMore": "https://aka.ms/finops/toolkit",
->   "exportScopes": <param:exportScopes>
+>   "exportScopes": <param:exportScopes>,
+>   "dataRetentionInMonths": 3
```

Comment:
@MSBrett You can use this for historical data backfill. The default is based on what most people do in the portal. I haven't looked to see if we have data for Power BI usage. Let me know if you think we should change the default duration or resolve this comment to ack.